src/esp/gfx/test/DepthUnprojectionTest.cpp

```diff
@@ -43,6 +43,7 @@ struct DepthUnprojectionTest : Mn::GL::OpenGLTester {
   void benchmarkGpuUnprojectExisting();
 };
 
+// NOLINTNEXTLINE(google-build-using-namespace)
 using namespace Mn::Math::Literals;
```

Comment:
Is there anything I can do on the engine side to not need to have this annotation in *every* source file that uses the literals? How does Clang Tidy behave with `using namespace std::chrono::literals`? Do you need to do the same or is there something that whitelists that namespace?